mkdocs_gen_files/config.py

```diff
@@ -0,0 +1,25 @@
+from mkdocs.config.config_options import File, OptionallyRequired
+
+
+class ListOfItems(OptionallyRequired):
```

Comment:
This one is copied from https://github.com/mkdocs/mkdocs/pull/2938 commit "Add generic ListOfItems config, refactor existing configs based on it"